integration-tests/src/main/java/io/druid/testing/ConfigFileConfigProvider.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Druid - a distributed column store.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.druid.testing;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.io.FileNotFoundException;
+import java.io.File;
+import java.io.IOException;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.util.Map;
+
+public class ConfigFileConfigProvider implements IntegrationTestingConfigProvider
+{
+  private String configFile;
+
+  private String routerHost = "";
+  private String brokerHost = "";
```

Comment:
noted.  The broker address is used by the current integration test code, and I think that shouldn't be changed under this PR.